dataikuapi/dss/recipe.py

```diff
@@ -1376,6 +1376,37 @@ def with_output_evaluation_store(self, mes_id):
         return self._with_output(mes_id, role="evaluationStore")
 
 
+class StandaloneEvaluationRecipeCreator(DSSRecipeCreator):
+    """
+    Builder for the creation of a new "Standalone Evaluate" recipe, from an
+    input dataset
+
+    .. code-block:: python
+
+        # Create a new standalone evaluation of a dataset
+
+        project = client.get_project("MYPROJECT")
+        builder = EvaluationRecipeCreator("my_standalone_evaluation_recipe", project)
```

Comment:
Typo here, I think you meant `StandaloneEvaluationRecipeCreator` 😄

Reply:
a classic copypasta mistake